core/src/test/scala/unit/kafka/metrics/MetricsTest.scala

```diff
@@ -113,7 +113,7 @@ class MetricsTest extends KafkaServerTestHarness with Logging {
     val initialReplicationBytesIn = meterCount(replicationBytesIn)
     val initialReplicationBytesOut = meterCount(replicationBytesOut)
     val initialBytesIn = meterCount(bytesIn)
-    val initialBytesOut = meterCount(bytesOut)
+//    val initialBytesOut = meterCount(bytesOut)
```

Comment:
Are these intentional?

Reply:
Previously this particular line causes the test to fail because `meterCount` doesn't work if `bytersOut` are not initialized